texts/deployment-guide.md

```diff
@@ -292,6 +292,20 @@ such as [Amazon RDS](https://aws.amazon.com/rds/) or [Google Cloud SQL](https://
 External databases
 will require the use of the [`use-external-dbs.yml`](/operations/use-external-dbs.yml) opsfile.
 
+The following databases are tested as part of the cf-deployment pipeline:
+- MySQL 8.0 using [pxc-release](https://github.com/cloudfoundry/pxc-release) as singleton and as Galera cluster
+- PostgreSQL 15 using [postgres-release](https://github.com/cloudfoundry/postgres-release)
+- GCP Cloud SQL for MySQL 8.0 as external database
+
+The following databases should work (not tested):
+- MySQL >=8.0
+- PostgreSQL >=12
```

Comment:
We might not want to use `>` here, since future versions of these databases could make breaking changes. For instance, I think we needed to do some work in various places to move from MySQL 5.7 -> MySQL 8.

Reply:
Makes sense. I added Postgres 11..14 as these versions have been used/tested in the past and there is no breaking change known (at least not to me).
I removed MySQL from the list of "should work" DBs since I don't have experience with it.